pom.xml

```diff
@@ -37,7 +37,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <sonar-maven-plugin.version>3.8.0.2131</sonar-maven-plugin.version>
     <spring-boot.version>2.7.5</spring-boot.version>
-    <h2.version>1.4.190</h2.version>
+    <h2.version>2.1.214</h2.version>
```

Comment:
Unfortunately CI fails because of this change. I recommend reverting this and submitting only the localization changes here so we can proceed with the merge.

Reply:
i revert the changes .